seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/KafkaSource.java

```diff
@@ -78,9 +78,9 @@ public List<CatalogTable> getProducedCatalogTables() {
     @Override
     public SourceReader<SeaTunnelRow, KafkaSourceSplit> createReader(
             SourceReader.Context readerContext) {
-
         BlockingQueue<RecordsWithSplitIds<ConsumerRecord<byte[], byte[]>>> elementsQueue =
-                new LinkedBlockingQueue<>();
+                new LinkedBlockingQueue<RecordsWithSplitIds<ConsumerRecord<byte[], byte[]>>>(
```

Comment:
```suggestion
                new LinkedBlockingQueue<>(
```

Reply:
Could you share the ci error link?

Reply:
I resubmitted the code again, and now it's okay.